### DISCUSSION TITLE
Controlling concurrency on a per file basis

### DISCUSSION THREAD

**Discussion by hbgl** [vitest-dev/vitest#2851] [Q&A] [answered]
I have multiple test files and most of them can be executed in parallel using threads. However, some of them I want to run single-threaded, i.e. without any other test files running at the same time. Is this possible with vitest?

**Comment by sheremet-va** [comment, marked as answer]
No, it's not possible

**Comment by sheremet-va** [comment]
You can open an enhancement issue if you want this to be supported, by the way
